packages/components/src/Notification/Notification.component.js

```diff
@@ -211,14 +220,14 @@ class NotificationsContainer extends React.Component {
 	}
 
 	onMouseEnter(event, notification) {
-		if (notification.error !== 'error' || this.props.autoLeaveError) {
+		if (notification.type !== TYPES.ERROR || this.props.autoLeaveError) {
```

Comment:
Assuming `.error` was a typo, given onMouseOut `conditions` and `notification` shape in propTypes